src/infra/dex/zeroex/mod.rs

```diff
@@ -19,7 +21,13 @@ pub struct ZeroEx {
     defaults: dto::Query,
 }
 
+/// https://0x.org/docs/introduction/0x-cheat-sheet#0x-contracts
+const DEFAULT_ALLOWANCE_TARGET: &str = "0x0000000000001fF3684f28c67538d4D072C22734";
```

Comment:
This address depends on the EVM hardfork at the time of deployment. I think we should probably future proof this and make this an optional config parameter which defaults to this value instead of having this always hardcoded constant here.